transcom/mymove
# B-18181 Deprecate Front End PPM Code

[B-18181](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871871)
[Integration PR](https://github.com/transcom/mymove/pull/11812)
### Summary

Remove PPM related front end code that is no longer used. I was quite thorough in investigating what code to remove, and making sure it does not break the application. All tests pass, but with 6000+ lines of code removed, I could've missed something.

### How to test

Login as a customer, create a new PPM, follow through the entire process (submit a move, approve with SC, submit move documents, review as closeout) and ensure that the app works as it should
Generally test throughout the application as much as possible.
